hbase-common/src/test/java/org/apache/hadoop/hbase/TestHBaseConfiguration.java

```diff
@@ -115,6 +115,19 @@ public void testSecurityConfCaseInsensitive() {
     conf.set("hbase.security.authentication", "KERBeros");
     Assert.assertTrue(User.isHBaseSecurityEnabled(conf));
   }
+  
+  @Test
+  public void testGetConfigOfShortcircuitRead() throws Exception {
+    Configuration conf = HBaseConfiguration.create();
+    Configuration.addDefaultResource("hdfs-default.xml");
+    assertEquals("hdfs-default.xml",
+            conf.getPropertySources("dfs.client.read.shortcircuit")[0]);
+    assertEquals("false", conf.get("dfs.client.read.shortcircuit"));
+    Configuration.addDefaultResource("hdfs-site.xml");
```

Comment:
Can you rename `hdfs-site` to be some other name, just to avoid other tests inadvertently picking up this configuration file, please?

Reply:
e.g. `hdfs-scr-enabled.xml`

Reply:
OK, I just want to simulate setting this value explicitly.Changing the name would work, too.